crox/src/main.rs

```diff
@@ -4,7 +4,7 @@ use std::io::BufWriter;
 use std::path::PathBuf;
 use std::time::{Duration, SystemTime, UNIX_EPOCH};
 
-use analyzeme::{ProfilingData, Timestamp};
+use analyzeme::{ProfilingData, Timestamp, Event as analyzemeEvent};
```

Comment:
It's kind of weird to have identifiers that start with lower case letters in Rust. Why don't we just do

```suggestion
use analyzeme::{self, ProfilingData, Timestamp};
```

and then

```rust
fn get_args(full_event: &analyzeme::Event) -> Option<FxHashMap<String, String>> {
```

below?

Reply:
done